packages/solana-client/src/GTransaction.ts

```diff
@@ -149,32 +152,41 @@ export namespace GTransaction {
         accounts: accounts.map((a) => a.address),
       })),
     };
+
+    return GTransaction.sign({
+      signers,
+      gtransaction,
+    });
   };
 
   export const sign = ({
-    secretKey,
+    signers,
     gtransaction,
   }: {
     gtransaction: GTransaction;
-    secretKey: Buffer | Uint8Array;
+    signers: Array<GKeypair>;
   }): GTransaction => {
-    const keypair = nacl.sign.keyPair.fromSecretKey(secretKey);
-    const address = bs58.encode(keypair.publicKey);
+    for (const { secretKey } of signers) {
+      const keypair = nacl.sign.keyPair.fromSecretKey(secretKey);
+      const address = bs58.encode(keypair.publicKey);
+
+      if (gtransaction.signatures.every((sig) => sig.address !== address)) {
```

Comment:
I wonder if we should have a flag to suppressing this error since it can sometimes be nice to pass in all possible signers (like when creating Stake transactions).

Reply:
I didn't have full context on how stake transactions are created/signed (I believe you're reffering to https://github.com/luma-team/luma/blob/38ceac8b7aab332b99230d73404ebe8058785bb9/server/src/utils/solana/stake-transaction-factory.ts#L277-L280), but I can add a flag suppressing this error! (will do)

Reply:
Added [`693a4ca` (#50)](https://github.com/glow-xyz/glow-js/pull/50/commits/693a4ca9535ce8bd912adc0da14b5559372b1d3c)